tests/Controller/ProductVariantApiTest.php

```diff
@@ -535,6 +535,41 @@ public function it_does_not_allow_nulling_information_about_product_variant_opti
         $this->assertResponseCode($response, Response::HTTP_BAD_REQUEST);
     }
 
+    /**
+     * @test
+     */
+    public function it_does_not_allow_nulling_information_about_product_variant_options_in_product_without_options(): void
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $productVariantsData = $this->loadFixturesFromFiles([
+            'resources/locales.yml',
+            'resources/product_variants.yml',
+        ]);
+
+        /** @var ProductInterface $product */
+        $product = $productVariantsData['product3'];
+
+        /** @var ProductVariantInterface $productVariant */
+        $productVariant = $productVariantsData['productVariant23'];
+
+        $version = $productVariant->getVersion();
+
+        $data =
+            <<<EOT
+        {
+            "version": "$version",
+            "optionValues": {
+                "MUG_TYPE": "MUG_TYPE_MEDIUM"
```

Comment:
during merge it struck me. Shouldn't this line be removed if the test is about nulling?
```suggestion
```

Reply:
Ah, I guess the test name is wrong, it should be "setting" not "nulling" 👍